src/libraries/System.Net.Security/tests/FunctionalTests/System.Net.Security.Tests.csproj

```diff
@@ -6,6 +6,7 @@
     <IgnoreForCI Condition="'$(TargetOS)' == 'browser'">true</IgnoreForCI>
     <EnablePreviewFeatures>true</EnablePreviewFeatures>
     <EventSourceSupport Condition="'$(TestNativeAot)' == 'true'">true</EventSourceSupport>
+    <DefineConstants>$(DefineConstants);NETWORKING_TESTS</DefineConstants>
```

Comment:
Is this also needed in the other networking tests? https://github.com/search?q=repo%3Adotnet%2Fruntime%20CertificateAuthority.cs&type=code says that the CA file gets included in Quic, Mail, Http, ...